.circleci/fastlane/Fastfile

```diff
@@ -10,15 +10,21 @@ lane :test do
 end
 
 def test_scheme(scheme)
-  scan(
-      workspace: '../ios/SalesforceReactTests/SalesforceReactTestApp.xcworkspace',
-      scheme: scheme,
-      device: ENV['DEVICE'] + ' (' + ENV['IOS_VERSION'] + ')',
-      output_directory: 'test_output',
-      output_types: 'html,junit',
-      code_coverage: true,
-      skip_build: true
-  )
+    $device.gsub!(' ', '-')
+    $ios.gsub!('.', '-')
+    system('xcrun simctl delete test_device') or true
+    sim_id = `xcrun simctl create test_device com.apple.CoreSimulator.SimDeviceType.#{$device} com.apple.CoreSimulator.SimRuntime.iOS-#{$ios}`.delete("\n")
+    destination = "test_device (" + `instruments -s devices | grep test_device | awk -F"[()]" '{print $2}'`.delete("\n") + ")"
```

Comment:
This fixes a really annoying issue with ios versioning.  13.2.x versions come out and we don't necessarily have to update the xcode image, however if we just specify 13.2 for the tests will fail since the devices are 13.2.2.  Next week it might be 13.2.3 and things would fail again.  My solution is to create a device and specify `13-2`, which will create a device with `13.2.2` today and `13.2.3` in the future if/when that happens.  I will send a pr for the main ios repo later today.